src/libstd/ffi/mod.rs

```diff
@@ -17,4 +17,8 @@ pub use self::c_str::CString;
 pub use self::c_str::c_str_to_bytes;
 pub use self::c_str::c_str_to_bytes_with_nul;
 
+pub use self::os_str::OsString;
+pub use self::os_str::OsStr;
```

Comment:
(not sure if we also want to add `IntoOsString` yet)

also raises questions as to why other conversion traits for `CString` don't exist yet, but they're backwards compatible to add!